README.md

```diff
@@ -47,14 +47,15 @@ shell or as entries in a `.env` file.
 
 - `EMAIL_ADDRESS`: the mail address you will use to send the emails
 - `RECIPIENT_ADDRESSES`: comma delimited list of recipient email addresses
-- `SUBJECT`: the email subject (could be `rss2email`)
+- `SUBJECT`: the email subject (could be `rss2email`). `$POST_COUNT` is replaced with the number of posts fetched.
```

Comment:
Not a huge fan of this, could you make post count a togglable flag like everything else?

Reply:
My thinking here is that that would not work with custom subjects in any way

If I wanted to change the subject to `Posts from Science Feeds: 12 Posts` I wouldn't be able to unless we had substitution variables.

Reply:
Wouldn't it work to have a flag that appends the number of posts to the subject? So you can set your subject to be "Science feeds" and if you enable the post count you get something like "Science feeds (42 posts)" or did I misunderstand something

Reply:
I can't really think of any other way myself so its fine for now :)